cibuildwheel/macos.py

```diff
@@ -233,6 +233,9 @@ def setup_python(python_configuration: PythonConfiguration,
     # inconsistencies if it's left unset.
     env.setdefault('MACOSX_DEPLOYMENT_TARGET', '10.9')
 
+    is_arm64 = python_configuration.identifier.endswith('arm64')
+    is_universal2 = python_configuration.identifier.endswith('universal2')
```

Comment:
I'm not sure about pulling these out... I kinda like how explicit/direct the code is currently. Unless there's a formatting reason why you'd like to do it?

Reply:
The reason is that when combined, they are really long and don't format well (and they are also used often).
```python
building_arm64 = is_arm64 or is_universal2
```
reads better than
```python
building_arm64 = python_configuration.identifier.endswith('arm64') or python_configuration.identifier.endswith('universal2') 
```

What about `config_is_*`?

Reply:
Pushed a version that uses config_is_arm64 and such.